tests/test_text_generation_example.py

```diff
@@ -43,7 +43,7 @@
             ("Qwen/Qwen1.5-7B", 4, False, 490.8621617893209, False),
             ("google/gemma-7b", 1, False, 109.70751574382221, True),
             ("google/gemma-2-9b", 1, False, 92.302359446567, True),
-            ("state-spaces/mamba-130m-hf", 1536, False, 5385.511100161605, False),
+            ("state-spaces/mamba-130m-hf", 1536, False, 19283.0330042467, False),
```

Comment:
could you enable check_output=true and add functional testing output to MODEL_OUTPUTS
```suggestion
            ("state-spaces/mamba-130m-hf", 1536, False, 19283.0330042467, True),
```

Reply:
does the test actually check the output,you didn't update MODEL_OUTPUTS dictionary with model and text output

Reply:
It can pass with setting True. But I think we don't need to change it this time.

Reply:
@zzhang37, you misunderstand me, test "passes" cause it does not enter the loop as model_name is not in MODEL_OUTPUTS keys, please re-read that part of the code- line 364.
we can remove that to make it `if check_output:` instead as devs might make the same mistake you did.

@regisss up to you and @jiminha  if you want to enforce functional testing but this is the reason it was added, to encourage devs to update with models.

Reply:
Yes I agree with @vidyasiv.
@zzhang37 You simply have to run the test with the `-s` flag to display the outputs. And then you can simply copy-paste the output of the model in the `MODEL_OUTPUTS` dictionary.

Reply:
Done.

Reply:
thanks @zzhang37